contrib/builder/rpm/fedora-23/Dockerfile

```diff
@@ -15,10 +15,10 @@ libtool \
 && set -x \
 && yum install -y $buildDeps \
 && export SECCOMP_PATH=$(mktemp -d) \
-&& git clone -b "$SECCOMP_VERSION" --depth 1 https://github.com/seccomp/libseccomp.git "$SECCOMP_PATH" \
+&& curl -fsSL "https://github.com/seccomp/libseccomp/releases/download/v${SECCOMP_VERSION}/libseccomp-${SECCOMP_VERSION}.tar.gz" \
+143			| tar -xzC "$SECCOMP_PATH" --strip-components=1 \
```

Comment:
the would be some copy pasta... you hungries

On Tue, Jan 19, 2016 at 5:44 PM, Tianon Gravi <notifications@github.com>
wrote:

> In contrib/builder/rpm/fedora-23/Dockerfile
> <https://github.com/docker/docker/pull/19377#discussion_r50204751>:
>
> > @@ -15,10 +15,10 @@ libtool \
> >  && set -x \
> >  && yum install -y $buildDeps \
> >  && export SECCOMP_PATH=$(mktemp -d) \
> > -&& git clone -b "$SECCOMP_VERSION" --depth 1 https://github.com/seccomp/libseccomp.git "$SECCOMP_PATH" \
> > +&& curl -fsSL "https://github.com/seccomp/libseccomp/releases/download/v${SECCOMP_VERSION}/libseccomp-${SECCOMP_VERSION}.tar.gz" \
> > +143			| tar -xzC "$SECCOMP_PATH" --strip-components=1 \
>
> Wait a minute, what's this 143? [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19377/files#r50204751>.
>